WordPress/src/main/java/org/wordpress/android/ui/people/PersonDetailFragment.java

```diff
@@ -2,22 +2,36 @@
 
 import android.animation.Animator;
 import android.animation.AnimatorInflater;
+import android.app.Activity;
+import android.app.AlertDialog;
 import android.app.Fragment;
+import android.content.Context;
+import android.content.DialogInterface;
 import android.os.Bundle;
 import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.LinearLayout;
+import android.widget.RadioButton;
 import android.widget.TextView;
 
 import org.wordpress.android.R;
+import org.wordpress.android.WordPress;
 import org.wordpress.android.datasets.PeopleTable;
+import org.wordpress.android.models.Account;
+import org.wordpress.android.models.AccountHelper;
+import org.wordpress.android.models.Blog;
+import org.wordpress.android.models.Capability;
 import org.wordpress.android.models.Person;
 import org.wordpress.android.util.AppLog;
 import org.wordpress.android.util.GravatarUtils;
 import org.wordpress.android.util.StringUtils;
 import org.wordpress.android.widgets.WPNetworkImageView;
 
-public class PersonDetailFragment extends Fragment {
+public class PersonDetailFragment extends Fragment implements View.OnClickListener {
```

Comment:
When first read this I though "Hmm, interesting, the fragment acts as a click handler? What does that respond to?!". I then understood that that's wired up to the Role row to trigger the role change dialog.

Can you share more info on this choice? If not a particular one, it's will probably be more clear to provide an dedicated `OnClickListener` to `setOnClickListener()` on the spot. Cheers!